README.md

```diff
@@ -19,6 +19,14 @@
 
   Some usage instructions
 
+    -npm run database
+      -creates the database
+    -npm run start
+      -starts your server/creates your table
+    -npm run seed
+      -seeds your database with fake data
+
```

Comment:
Great idea updating the Readme regularly! I'm gonna do that too :)

Reply:
+1!